reference/providers/ecs.md

```diff
@@ -7,13 +7,41 @@ sidebar:
 
 {% include toc %}
 
-## Resource Mapping
+## Resource mapping
 
 ### Server Group
```

Comment:
Probably someone like @dorbin can help us with correct casing here.  This change has "Resource mapping" and "Operation mapping", but "Server Group" (capital G)

Reply:
In [several](https://www.spinnaker.io/reference/providers/cf/) [other](https://www.spinnaker.io/reference/providers/kubernetes/) [reference](https://www.spinnaker.io/reference/providers/appengine/) pages, section headers (e.g., "Resource mapping") are cased as above. "Server Group" is more inconsistent across these same pages, however is either always or mostly capitalized on both words. 

I'll change "Server Group" to always be upper case, but defer to guidance from @dorbin  or others.